lib/ldclient-rb/ldclient.rb

```diff
@@ -387,7 +387,7 @@ def evaluate_internal(key, context, default, with_reasons)
 
       context = Impl::Context::make_context(context)
       unless context.valid?
-        @config.logger.error { "[LDClient] Context was invalid for flag evaluation (#{context.error}); returning default value" }
+        @config.logger.error { "[LDClient] Context was invalid for flag evaluation of key '#{key}' (#{context.error}); returning default value" }
```

Comment:
What do you think about this alternative language?

```suggestion
        @config.logger.error { "[LDClient] Context was invalid for evaluation of flag '#{key}' (#{context.error}); returning default value" }
```